wls-vorfaelleundvorkommnisse-service/src/main/java/de/muenchen/oss/wahllokalsystem/vorfaelleundvorkommnisseservice/service/EreignisModel.java

```diff
@@ -0,0 +1,9 @@
+package de.muenchen.oss.wahllokalsystem.vorfaelleundvorkommnisseservice.service;
+
+import de.muenchen.oss.wahllokalsystem.vorfaelleundvorkommnisseservice.domain.ereignis.Ereignisart;
```

Comment:
bitte aus dem eigenen Package verwenden, so koppeln wir uns an die Datenhaltung

Reply:
siehe commit [4aba2e4](https://github.com/it-at-m/Wahllokalsystem/pull/412/commits/4aba2e455e97ffe7020a8fe113ac957627caad41)